apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -35,6 +35,8 @@ function getTourAssets( key ) {
 }
 
 function getTourSteps( localeSlug, referencePositioning ) {
+	const paymentBlockSupportUrl = `https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/#how-to-use-the-payments-block-video`;
```

Comment:
Isn't there a function in Calypso that can determine the correct i18n support link? We have support docs in many languages (with different URL formats), and I _think_ there's some way that will figure it out for us.

Reply:
👍